docs/events-challenges/community-events.md

```diff
@@ -0,0 +1,47 @@
+# Community Events
+
+Community events are a great way to connect with other players, showcase your creativity, and enjoy everything Theatria has to offer as a community. Events often start in Discord, and everyone is encouraged to participate or contribute their ideas.
+
+## How to Get Involved
+
+### Suggest an Event
+- **Join our Discord**: Start by joining Theatria's Discord server if you haven't already. This is where event planning begins.
```

Comment:
Maybe add a link here as well, since we tell them to join the discord, could be helpful to let them click the word here and join.